activate-virtualenv.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# ==========================================================================
+# Script for activating the virtualenv (creating it first, if necessary).
+# NOTE: Run this script while in the project root directory.
+#       It will not run correctly when run from another directory.
+# ==========================================================================
+
+# Set script to exit on any errors.
+set -e
+
+# Presumes the varialble VIRTUAL_ENV is already set to the name of the venv
+
+echo 'Activating virtualenv, if not already activated...'
+
+source $(brew --prefix)/bin/virtualenvwrapper.sh
+
+# Activate virtualenv, if not already activated
+if ! type "$workon" &>/dev/null; then
```

Comment:
This code seems to have been moved from [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/.env_SAMPLE#L132), but this check seems obsolete now that virtualenvwrapper is always invoked above (if that's the case, `workon` should always be available).

This code seems to have a slight error -- it should say `type "workon"` not `type "$workon"` as the former checks whether `workon` exists (what we want) and the latter checks the value of the environment variable `$workon`. If this variable isn't set (I couldn't find any documentation suggesting that virtualenv or virtualenvwrapper sets this), this test will always pass and setup or activate the virtualenv - not a problem in practice but still extra logic that isn't doing anything.